implementations/micrometer-registry-dynatrace/src/main/java/io/micrometer/dynatrace/DynatraceConfig.java

```diff
@@ -76,8 +85,37 @@ default DynatraceApiVersion apiVersion() {
                 .orElse(DynatraceApiVersion.V1);
     }
 
+    default String metricKeyPrefix() {
+        // returns an empty string if nothing is set in the properties.
+        return getString(this, "metricKeyPrefix").orElse("");
+    }
+
+    @Nullable
+    default Map<String, String> defaultDimensions() {
+        return Collections.emptyMap();
```

Comment:
This seems to be incomplete.

Reply:
This is just the default implementation, which seems to be overwritten by the `DynatracePropertiesConfigAdapter` in Spring Boot. By default I think it makes sense to return an empty map, this method can always be overwritten by users of this method and extended to include default dimensions of their liking.